### ISSUE TITLE
Sub-optimal defaults in "make serve"

### ISSUE THREAD

**Issue by bep** [kubernetes/website#9967]
See https://github.com/kubernetes/website/blob/master/Makefile#L20

```bash
serve: ## Boot the development server.
        hugo server --ignoreCache --disableFastRender
```

I'm not sure who uses `make serve`, but if it is content-editors, the above makes for a poor (very slow) default.

My favourites:

For content editing:

```bash
  hugo server --navigateToChanged
```

For design work:

```bash
  hugo server
 # And some times
 hugo server --disableFastRender
```

Note that I have left out ` --ignoreCache`, which is really aggressive. I prefer to use the "disable cache" check box in Chrome developer tools.

**Comment by DanyC97** [comment]
@lucperkins your thoughts ?

**Comment by lucperkins** [comment]
`--navigateToChanged` might be worth trying at some point but could introduce problems for people if introduced out of the blue. I do agree that `--ignoreCache` is overly aggressive. I'll remove it.

**Comment by DanyC97** [comment]
fixed by #13133
